go.mod

```diff
@@ -1,15 +1,15 @@
 module github.com/ChainSafe/gossamer
```

Comment:
Can you open a PR with those dependency changes separately perhaps?
To keep a commit log making sense 😉

Reply:
Probably also rebase on development since a bunch of deps were bumped on development (👀 centrifuge etc.)